products/bash/register-tracing.sh

```diff
@@ -14,6 +14,7 @@
 #
 # PARAMETERS:
 #   -n : <namespace> (string), Defaults to "cp4i"
+#   -e : <test_environment> (flag), Defaults to "false"
```

Comment:
Does this script get called somewhere with this parameter?

Reply:
yes this called from install.sh but that is to register product in main-namespace. So what is the problem here then?

Reply:
I think it's only called further down during the release-mq script, it passes in -e parameter there as part of running for a 'test' namespace.